rocAL/rocAL/include/api/rocal_api_augmentation.h

```diff
@@ -45,10 +45,20 @@ extern "C"  RocalImage  ROCAL_API_CALL rocalSequenceRearrange(RocalContext conte
 /// \param dest_width
 /// \param dest_height
 /// \param is_output
+/// \param scaling_mode The resize scaling_mode to resize the image.
+/// \param max_size Limits the size of the resized image.
+/// \param resize_shorter The length of the shorter dimension of the image.
+/// \param resize_longer The length of the larger dimension of the image.
+/// \param interpolation_type The type of interpolation to be used for resize.
 /// \return
 extern "C"  RocalImage  ROCAL_API_CALL rocalResize(RocalContext context, RocalImage input,
-                                                unsigned dest_width, unsigned dest_height,
-                                                bool is_output );
+                                                   unsigned dest_width, unsigned dest_height,
+                                                   bool is_output,
+                                                   RocalResizeScalingMode scaling_mode = ROCAL_SCALING_MODE_STRETCH,
+                                                   std::vector<unsigned> max_size = {},
+                                                   unsigned resize_shorter = 0,
```

Comment:
instead of passing 2 separate parameters for resize_shorted and resize_longer, we can use a bitmask where each bit can represent the flag. That way it is easy to extend for future modes and reduces an argument

Reply:
RPP accepts resize width and height as unsigned integers. If we change the type of dst_width(resize_x), dest_height(resize_y), resize_shorter and resize_longer to float in order to match with DALI resize API we may need to change the way RPP accepts the inputs too